### PR TITLE
[Issue #79] Add --keep-job-dir option and delete the job directory by default.

### PR DESCRIPTION
**Pull request by calebevans**

### CODE REVIEW

File: docs/cli_usage_guide.md [hunk 1]
```diff
@@ -50,19 +50,23 @@ Many of the arguments for this command have set defaults or will use an environm
 Usage: firewatch report [OPTIONS]
 
 Options:
+  --keep-job-dir                If set, firewatch will not delete the job
```
Comment (oharan2) [inline docs/cli_usage_guide.md:53]: Might want to consider:
`--keep-job-dir BOOL`

`--fail_with_test_failures BOOL`

Comment (calebevans) [reply docs/cli_usage_guide.md]: I have updated, this is just the way click formats their help output.

**Pull request merged**